src/main/java/sirius/biz/codelists/IDBLookupTable.java

```diff
@@ -272,14 +272,30 @@ protected <T> T makeObject(Class<T> type, ObjectNode jsonData) {
     }
 
     @Override
-    protected Stream<LookupTableEntry> performSuggest(Limit limit, String searchTerm, String language) {
+    protected Stream<LookupTableEntry> performSuggest(Limit limit,
+                                                      String searchTerm,
+                                                      String language,
+                                                      boolean considerDeprecatedValues) {
         try {
+            if (considerDeprecatedValues) {
+                return table.query()
+                            .searchInAllFields()
+                            .searchValue(searchTerm)
+                            .translate(language)
+                            .manyRows(limit, codeField, nameField, descriptionField)
+                            .map(row -> new LookupTableEntry(row.at(0).asString(),
+                                                             row.at(1).asString(),
+                                                             row.at(2).getString()));
+            }
```

Comment:
**Optional:** I think it would be nice to add the caveat/reason for this if-statement (that is mentioned in the commit) as a small comment to prevent accidental breakage via refactoring/cleanup